pkg/storage/driver/driver.go

```diff
@@ -17,18 +17,18 @@ limitations under the License.
 package driver // import "k8s.io/helm/pkg/storage/driver"
 
 import (
-	"errors"
+	"fmt"
 
 	rspb "k8s.io/helm/pkg/proto/hapi/release"
 )
 
 var (
 	// ErrReleaseNotFound indicates that a release is not found.
-	ErrReleaseNotFound = errors.New("release: not found")
+	ErrReleaseNotFound = func(release string) error { return fmt.Errorf("release: '%s' not found", release) }
```

Comment:
These should be using `%q` instead of `'%s'` so you don't have to quote it yourself